app/editor/src/features/admin/evening-overview/interfaces/IEveningOverviewItemType.ts

```diff
@@ -0,0 +1,12 @@
+import { OptionItem } from 'tno-core';
+
+export enum EveningOverviewItemType {
+  Intro = 'Intro',
+  Story = 'Story',
+  Ad = 'Ad',
+}
+export const eveningOverviewItemTypeOptions = [
```

Comment:
Try not to hide separate features or tools in the same file as something else.  Also don't place OptionItems in the interfaces folder.  We normally keep things like this in utilities.  

Additionally I thought we had a util that automatically converts enum values into an array of OptionItems.